e2e/cypress/integration/menus/status_dropdown_spec.js

```diff
@@ -28,163 +35,106 @@ describe('Status dropdown menu', () => {
         cy.reload();
     });
 
-    it('Displays default menu when status icon is clicked', () => {
-        // # Wait for posts to load
-        cy.get('#postListContent', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible');
-
-        // # Click status menu
-        cy.get('.MenuWrapper .status-wrapper.status-selector button.status').click();
-
-        // # Wait for status menu to transition in
-        cy.get('.MenuWrapper.status-dropdown-menu .Menu__content.dropdown-menu').should('be.visible');
+    after(() => {
+        // # Reset to enable the custom status
+        cy.apiUpdateConfig({TeamSettings: {EnableCustomUserStatuses: true}});
     });
 
-    it('Changes status icon to online when "Online" menu item is selected', () => {
-        // # Wait for posts to load
-        cy.get('#postListContent').should('be.visible');
-
-        // # Set user status to away to ensure menu click changes status
-        cy.apiUpdateUserStatus('away').then(() => {
-            // # Click status menu
-            cy.get('.MenuWrapper .status-wrapper.status-selector button.status').click();
-
-            // # Wait for status menu to transition in
-            cy.get('.MenuWrapper.status-dropdown-menu .Menu__content.dropdown-menu').should('be.visible');
+    it('MM-T2927_1 Should open status menu', () => {
+        // # Open status menu
+        openStatusMenu(TIMEOUTS.ONE_MIN);
```

Comment:
Please use the existing `cy.uiOpenSetStatusMenu`. Same should be applied to other instances.
```suggestion
        cy.uiOpenSetStatusMenu();
```